src/providers/ipa/ipa_init.c

```diff
@@ -669,6 +669,13 @@ static errno_t ipa_init_misc(struct be_ctx *be_ctx,
         return ENOMEM;
     }
 
+    ret = sdap_init_certmap(sdap_id_ctx, sdap_id_ctx);
+    if (ret != EOK) {
+        DEBUG(SSSDBG_CRIT_FAILURE,
+              "Failed to initialized certificate mapping.\n");
```

Comment:
Can you print the retval here? (`sdap_init_certmap` doesn't print the errors either so it might not be easy to see why the code failed)

Reply:
I added error code and message in sdap_setup_certmap() so that the error can be better related to rule which caused the failure.